requirements.txt

```diff
@@ -14,3 +14,4 @@ linkchecker==10.*
 # extra dependencies required by specific notebooks
 # used by notebooks in Examples/Pecos/
 pecos>=0.2.0
+tensorflow==2.8.0
```

Comment:
I've only tested the notebooks for `tensorflow 2.8.0` since that is what I have installed locally. @lbianchi-lbl is there a specific (version 2.5.0, 2.7.0, ...) you would suggest using as a minimum version and could the version specified here be the cause of the failing build tests?

Reply:
First and foremost, should we be specifying a version for tensorflow at all?

Reply:
@andrewlee94 pinning the Tensorflow version was my suggestion. My reasoning is that it makes our builds more reproducible, without any significant downside for users (since they would likely use `idaes get-examples` and not `requirements.txt` directly).